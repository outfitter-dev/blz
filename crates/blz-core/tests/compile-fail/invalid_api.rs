fn main() {
    let _ = blz_core::this_function_does_not_exist();
}

