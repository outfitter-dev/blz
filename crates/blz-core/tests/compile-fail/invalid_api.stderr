error[E0425]: cannot find function `this_function_does_not_exist` in crate `blz_core`
 --> tests/compile-fail/invalid_api.rs:2:23
  |
2 |     let _ = blz_core::this_function_does_not_exist();
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `blz_core`
